src/components/Editor/AdditionalInformation/DataProcesses.vue

```diff
@@ -0,0 +1,227 @@
+              <v-card-title class="green white--text">
+                Add a data process
+              </v-card-title>
+              <v-card-text>
+                <v-container fluid>
+                  <v-row justify="start">
+                    <v-col class="col-6">
+                      <v-select
+                        v-model="newDataProcess.type"
+                        :items="dataProcessTypes"
+                        menu-props="auto"
+                        label="Type"
+                        :rules="[rules.isRequired()]"
+                        outlined
+                        return-object
+                      />
+                    </v-col>
+                    <v-col class="col-6">
+                      <v-text-field
+                        v-model="newDataProcess.name"
+                        label="Name"
+                        :rules="[rules.isRequired()]"
+                        outlined
+                      />
+                    </v-col>
+                    <v-col class="col-6">
+                      <v-text-field
+                        v-model="newDataProcess.url"
+                        label="URL"
+                        :rules="[rules.isUrl()]"
+                        outlined
+                      />
+                    </v-col>
+                  </v-row>
+                </v-container>
+              </v-card-text>
+              <v-card-actions>
+                <v-btn
+                  class="green white--text"
+                  :disabled="!formValid"
+                  @click="addDataProcess()"
+                >
+                  Add Access Point
+                </v-btn>
+                <v-btn
+                  class="red white--text"
+                  @click="openEditor = false"
+                >
+                  Cancel
+                </v-btn>
+              </v-card-actions>
+            </v-card>
+
+          <!-- new access point dialogue ends -->
+          </v-row>
+        </v-container>
+      </v-form>
+    </v-dialog>
+  </v-card>
+</template>
+
+<script>
+import {mapGetters, mapState} from "vuex";
+import {isRequired, isUrl} from "@/utils/rules.js";
+import {compareArray} from "@/utils/utils.js";
+
+export default {
+  name: "DataProcesses",
+  data() {
+    return {
+      openEditor: false,
+      formValid: false,
+      newDataProcess: {},
+      rules: {
+        isRequired: function(){return isRequired()},
+        isUrl: function(){return isUrl()},
+      }
+    }
+  },
+  computed: {
+    ...mapGetters("record", ["getSection", "getChanges"]),
+    ...mapState("editor", ["dataProcessTypes"]),
+    section(){
+      return this.getSection('additionalInformation');
+    },
+    initialFields(){
+      return this.getSection("additionalInformation").initialData
+    },
+    currentFields(){
+      let data = this.getSection("additionalInformation").data
+      if (!data.data_processes) data.data_processes = [];
+      return data;
```

Comment:
Compact version (to be tested)
```suggestion
      return this.getSection("additionalInformation").data || [];
```